### PR TITLE
[ML][Pipelines] Add missing properties in schedule schema

### PR DESCRIPTION
**Pull request by brynn-code**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [x] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [x] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/ml/azure-ai-ml/azure/ai/ml/_schema/schedule/schedule.py [hunk 1]
```diff
@@ -40,3 +40,4 @@ class ScheduleSchema(ResourceSchema):
     creation_context = NestedField(CreationContextSchema, dump_only=True)
     is_enabled = fields.Boolean(dump_only=True)
     provisioning_state = fields.Str(dump_only=True)
+    properties = fields.Dict(keys=fields.Str(), values=fields.Str(allow_none=True))
```
Comment (wangchao1230) [inline sdk/ml/azure-ai-ml/azure/ai/ml/_schema/schedule/schedule.py:43]: Do we support nested dict in properties?

**Review by wangchao1230 — APPROVED** [review]

File: sdk/ml/azure-ai-ml/azure/ai/ml/_schema/schedule/schedule.py (hunk 1 above)
Comment (brynn-code) [reply sdk/ml/azure-ai-ml/azure/ai/ml/_schema/schedule/schedule.py:43]: I remember that's not supported

Comment (brynn-code) [reply sdk/ml/azure-ai-ml/azure/ai/ml/_schema/schedule/schedule.py:43]: Nested dict in sdk will become string after submission, while nested dict in cli is not supported and will raise validation error, user should write string manually.
image
image

Comment (wangchao1230) [reply sdk/ml/azure-ai-ml/azure/ai/ml/_schema/schedule/schedule.py:43]: seems good for now. Let's revisit in future.

**Pull request merged**